### ISSUE TITLE
A feature idea - 3D on 3DS?

### ISSUE THREAD

**Issue by Cee123** [libretro/beetle-vb-libretro#54]
I noticed on the emulator Red Dragon for the 3DS that there's a feature that actually allows virtual boy games to be run in 3D which I thought was pretty interesting. But I'm not sure but it looks like it takes advantage of the 3D on the 3DS.

**Comment by RealCreepyJeff** [comment]
It does. I would love to see this implemented in beetle-vb! Also technically it's r3Ddragon

**Comment by Cee123** [comment]
> 
> 
> It does. I would love to see this implemented in beetle-vb! Also technically it's r3Ddragon

Yeah, well, r3Ddragon that's what I meant. lol There's also a 3D feature on the PC version but I could not get the emulator to work on PC at all. Haha

**Comment by HeyItsLollie** [comment]
I tried Beetle-VB on 3DS for this exact reason, it's surprising that this feature isn't already implemented. Retroarch's menu even takes advantage of the 3D screen, but just as a visual effect and not for the game itself.

**Comment by Syko7956** [comment]
Hope it can be done. Unfortunately it run poorly on N3DS. When r3Ddragon shows that it can run better. At least back when it used to work 5 years ago. I have noticed this core improved since it used to crash on boot a year ago. Hoping it gets more optimized

**Comment by gingerbeardman** [comment]
Came here to post this feature request. Works be great to see these games in 3D.
